challenge_3/python/returnlove/src/majority_element.py

```diff
@@ -0,0 +1,28 @@
+"""
+To find the majority elements in a list.
+	Counts number of times each element is present in the list
+	prints all the elements which are present more than half of the list
+"""
+
+import math
+
+def find_majority_element(input_list):
+	majority_condition = math.floor(len(input_list)/2)
+	print(majority_condition)	
+	my_dict = {}
+	for element in input_list:
+		if element not in my_dict.keys():
+			my_dict[element] = 1
+		else:
+			my_dict[element] += 1
+
+	for k in my_dict.keys():
+		if my_dict[k] > majority_condition:
+			print(k)
```

Comment:
@hkl0902 got that, it makes easy to unit test. Here is my summary. So basically, you were referring to avoid printing the result directly in the function but we can use the print statement when calling the function _(print(do_something())_, because we need to see the result on console.